data/transactions/verify/txn_test.go

```diff
@@ -61,11 +70,12 @@ func generateTestObjects(numTxs, numAccs int, blockRound basics.Round) ([]transa
 		addresses[i] = addr
 	}
 	var iss, exp int
+	u := uint64(0)
 	for i := 0; i < numTxs; i++ {
 		s := rand.Intn(numAccs)
 		r := rand.Intn(numAccs)
 		a := rand.Intn(1000)
-		f := config.Consensus[protocol.ConsensusCurrentVersion].MinTxnFee + uint64(rand.Intn(10))
+		f := config.Consensus[protocol.ConsensusCurrentVersion].MinTxnFee + uint64(rand.Intn(10)) + u
```

Comment:
this was done to ensure that we don't end up with identical transactions when generating large number of txns.